### PR TITLE
Fbsetup

### PR DESCRIPTION
**Pull request by Yhugzee**

### CODE REVIEW

File: backend/src/firebase.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: FIREBASE_API_KEY,
+  authDomain: "origins-digital.firebaseapp.com",
+  projectId: "origins-digital",
+  storageBucket: "origins-digital.appspot.com",
+  messagingSenderId: "898956315661",
+  appId: "1:898956315661:web:8d467e832ba230ef295944",
+};
```
Comment (rocambille) [inline backend/src/firebase.js:16]: why do you need firebase ? and are you sure these ids can be seen into a public repository ?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: backend/src/firebase.js (hunk 1 above)
Comment (Yhugzee) [reply backend/src/firebase.js:16]: I need firebase to store the videos outside the backend as requested by the client and I have hidden the most important thing which is my API key

Comment (rocambille) [reply backend/src/firebase.js:16]: even if storing outside the backend was mentionned in the product backlog, in the context of the training you will use the same server for storing the videos : so no need of firebase

Comment (Yhugzee) [reply backend/src/firebase.js:16]: OK

**Pull request closed without merging**